tests/suites/constraints/constraints.sh

```diff
@@ -0,0 +1,60 @@
+run_constraints_aws() {
+  # Echo out to ensure nice output to the test suite.
+  echo
+
+  file="${TEST_DIR}/constraints-aws.txt"
+
+  ensure "constraints-aws" "${file}"
+
+	echo "Deploy 5 machines with different constraints"
+  juju add-machine --constraints "root-disk=16G"
+  juju add-machine --constraints "cores=4"
+  juju add-machine --constraints "cpu-power=30"
+  juju add-machine --constraints "root-disk=16G cpu-power=30"
```

Comment:
I think you can drop tests for 'cpu-power'. I've never heard of anyone using it in anger, and it only works on AWS.
I think it is reasonable to have a test that does root-disk + cores, though.

Reply:
done